src/zzz_od/application/hollow_zero/lost_void/operation/lost_void_run_level.py

```
@@ -316,35 +317,34 @@ def try_interact(self) -> OperationRoundResult:
-                if current_interact_target is not None:
-                    self.interact_target = current_interact_target
-
-                self.ctx.controller.interact(press=True, press_time=0.2, release=True)
-                return self.round_wait('交互', wait=1)
-
-        if not self.ctx.lost_void.in_normal_world(self.last_screenshot):  # 按键消失 说明开始加载了
+            area = self.ctx.screen_loader.get_area('迷失之地-大世界', '区域-交互文本')
+            part = cv2_utils.crop_image_only(self.last_screenshot, area.rect)
+            ocr_result_map = self.ctx.ocr.run_ocr(part)
+            current_interact_target = None
+            for ocr_result in ocr_result_map.keys():
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)